tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerTests.cs

```diff
@@ -198,43 +195,42 @@ public async Task QnaMaker_TraceActivity()
                 await context.SendActivityAsync("echo:" + context.Activity.Text);
             })
                 .Send("how do I clean the stove?")
-                    .AssertReply((activity) => Assert.AreEqual(activity.Type, ActivityTypes.Typing))
+                    .AssertReply((activity) => Assert.Equal(activity.Type, ActivityTypes.Typing))
                     .AssertReply("echo:how do I clean the stove?")
                 .Send("bar")
-                    .AssertReply((activity) => Assert.AreEqual(activity.Type, ActivityTypes.Typing))
+                    .AssertReply((activity) => Assert.Equal(activity.Type, ActivityTypes.Typing))
                     .AssertReply("echo:bar")
                 .StartTestAsync();
 
             // Validate Trace Activity created
             var pagedResult = await transcriptStore.GetTranscriptActivitiesAsync("test", conversationId);
-            Assert.AreEqual(7, pagedResult.Items.Length);
-            Assert.AreEqual("how do I clean the stove?", pagedResult.Items[0].AsMessageActivity().Text);
-            Assert.IsTrue(pagedResult.Items[1].Type.CompareTo(ActivityTypes.Trace) == 0);
+            Assert.Equal(7, pagedResult.Items.Length);
+            Assert.Equal("how do I clean the stove?", pagedResult.Items[0].AsMessageActivity().Text);
+            Assert.True(pagedResult.Items[1].Type.CompareTo(ActivityTypes.Trace) == 0);
             var traceInfo = ((JObject)((ITraceActivity)pagedResult.Items[1]).Value).ToObject<QnAMakerTraceInfo>();
-            Assert.IsNotNull(traceInfo);
-            Assert.IsNotNull(pagedResult.Items[2].AsTypingActivity());
-            Assert.AreEqual("echo:how do I clean the stove?", pagedResult.Items[3].AsMessageActivity().Text);
-            Assert.AreEqual("bar", pagedResult.Items[4].AsMessageActivity().Text);
-            Assert.IsNotNull(pagedResult.Items[5].AsTypingActivity());
-            Assert.AreEqual("echo:bar", pagedResult.Items[6].AsMessageActivity().Text);
+            Assert.NotNull(traceInfo);
+            Assert.NotNull(pagedResult.Items[2].AsTypingActivity());
+            Assert.Equal("echo:how do I clean the stove?", pagedResult.Items[3].AsMessageActivity().Text);
+            Assert.Equal("bar", pagedResult.Items[4].AsMessageActivity().Text);
+            Assert.NotNull(pagedResult.Items[5].AsTypingActivity());
+            Assert.Equal("echo:bar", pagedResult.Items[6].AsMessageActivity().Text);
             foreach (var activity in pagedResult.Items)
             {
-                Assert.IsTrue(!string.IsNullOrWhiteSpace(activity.Id));
-                Assert.IsTrue(activity.Timestamp > default(DateTimeOffset));
+                Assert.True(!string.IsNullOrWhiteSpace(activity.Id));
```

Comment:
Maybe we could change it to something like this.
```suggestion
                Assert.False(string.IsNullOrWhiteSpace(activity.Id));
```

Reply:
Done. Thanks!